simplified-tests/src/test/resources/org/nypl/simplified/tests/books/profile-rates.json

```diff
@@ -0,0 +1,22 @@
+{
+  "@version" : 20200504,
+  "displayName" : "",
+  "preferences" : {
+    "showTestingLibraries" : true,
+    "hasSeenLibrarySelectionScreen" : true,
+    "useExperimentalR2" : true,
+    "showDebugSettings" : true,
+    "mostRecentAccount" : "5310437f-db1a-492e-a09f-1ceaa43303dd",
+    "readerPreferences" : {
+      "font_scale" : 100.0,
+      "font_family" : "READER_FONT_OPEN_DYSLEXIC",
+      "color_scheme" : "SCHEME_WHITE_ON_BLACK"
+    },
+    "playbackRates" : "{\"bookid1\": \"NORMAL_TIME\", \"bookid2\": \"ONE_AND_A_QUARTER_TIME\"}",
```

Comment:
This seems strange to me. Instead of storing a string of JSON, can we make this a JSON object, like `readerPreferences` is? For example:

```
"playbackRates" : {
  "bookid1" : "NORMAL_TIME", 
  "bookid2" : "ONE_AND_A_QUARTER_TIME"
},
```